aptos-move/e2e-testsuite/src/tests/failed_transaction_tests.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_gas_meter::{StandardGasAlgebra, StandardGasMeter};
+use aptos_gas_schedule::{AptosGasParameters, LATEST_GAS_FEATURE_VERSION};
+use aptos_language_e2e_tests::{common_transactions::peer_to_peer_txn, executor::FakeExecutor};
+use aptos_memory_usage_tracker::MemoryTrackedGasMeter;
+use aptos_types::{
+    state_store::{state_key::StateKey, TStateView},
+    transaction::ExecutionStatus,
+    vm_status::{StatusCode, VMStatus},
+    write_set::WriteOp,
+};
+use aptos_vm::{data_cache::AsMoveResolver, transaction_metadata::TransactionMetadata, AptosVM};
+use aptos_vm_logging::log_schema::AdapterLogSchema;
+use aptos_vm_types::storage::StorageGasParameters;
+use claims::assert_some;
+use move_core_types::vm_status::StatusCode::TYPE_MISMATCH;
+
+#[test]
+fn failed_transaction_cleanup_test() {
+    let mut executor = FakeExecutor::from_head_genesis();
+    // TODO(Gas): double check this
```

Comment:
what to double check?

Reply:
removed this file. The tests have been refactored to other files